requirements.txt

```diff
@@ -0,0 +1,7 @@
+black==20.8b1
```

Comment:
Lets order this list and add an empty line between each package to try to help git better detect conflicts.

For that matter, we can add a comment to the top of the file...

```suggestion
# Keep this list in alphabetical order, with a single blank line between each package
black==20.8b1
```